### ISSUE TITLE
Installing 34 on Travis fails

### ISSUE THREAD

**Issue by wpoely86** [luaposix/luaposix#276]
For example: https://travis-ci.org/wpoely86/Lmod/jobs/229668239
This is with Lua 5.3.4 and luarocks 2.4.2
```
$ luarocks install luaposix

Installing https://luarocks.org/luaposix-34.0-1.src.rock

build-aux/luke package="luaposix" version="34.0-1" PREFIX="/home/travis/lua_install/lib/luarocks/rocks/luaposix/34.0-1" LUA="/home/travis/lua_install/bin/lua" LUA_INCDIR="/home/travis/lua_install/include" CFLAGS="-O2 -fPIC" LIBFLAG="-shared" LIB_EXTENSION="so" OBJ_EXTENSION="o" INST_LIBDIR="/home/travis/lua_install/lib/luarocks/rocks/luaposix/34.0-1/lib" INST_LUADIR="/home/travis/lua_install/lib/luarocks/rocks/luaposix/34.0-1/lua"

lua: build-aux/luke:120: attempt to concatenate a nil value (local 'found')

stack traceback:

	build-aux/luke:120: in upvalue 'check_executable_in_path'

	build-aux/luke:197: in local 'block'

	build-aux/luke:432: in function 'std.functional.dropuntil'

	build-aux/luke:197: in function <build-aux/luke:197>

	(...tail calls...)

	build-aux/luke:432: in function 'std.functional.dropuntil'

	build-aux/luke:210: in function <build-aux/luke:210>

	(...tail calls...)

	build-aux/luke:220: in function 'luke.configure.config_ldoc'

	build-aux/luke:244: in upvalue 'build_modules'

	build-aux/luke:254: in function 'luke.main'

	build-aux/luke:618: in main chunk

	[C]: in ?
```

**Comment by gvvaughan** [comment]
thanks for the heads up; I think this is a simple matter of removing the ldoc references from the release lukefile, if you want to try patching while you wait for me to get back to a computer and fix it later today.

alternatively, please depend on v33.4.0 for a short while.

I'll follow up here after I push a fix.

**Comment by gvvaughan** [comment]
I believe this issue is fixed in git-master now.

I've also pulled the 34.0 release from luarocks, pending some enhancements needed to get luaposix with luke building on macOS Yosemite.  Please open a new issue if you encounter this again (or any other problems).

**Comment by wpoely86** [comment]
Yes, it's fixed. Thanks.
